### PR TITLE
Add aria-label to display current element for screen-readers

### PR DESCRIPTION
**Pull request by woofers**
Adds a aria-label with the value of the current element being typed.

This will help screen-readers better understand the content
rather than re-reading the screen each time a new letter is added.

### CODE REVIEW

File: lib/ReactRotatingText.js [hunk 1]
```diff
@@ -200,9 +218,10 @@ ReactRotatingText.defaultProps = {
   deletingInterval: 50,
   emptyPause: 1000,
   eraseMode: 'erase',
-  items: ['first', 'second', 'third'],
+  items: ['first', 'second', 'third', 'fourth', 'fifth'],
```
Comment (charj) [inline lib/ReactRotatingText.js]: Were these values intentionally updated? 
Not that it would cause any problems but maybe should be a separate change if this is what we want :+1:

File: lib/ReactRotatingText.js [hunk 2]
```diff
@@ -155,21 +170,23 @@ var ReactRotatingText = (function (_React$Component) {
...
-      var cursor = _props2.cursor;
-      var deletingInterval = _props2.deletingInterval;
-      var emptyPause = _props2.emptyPause;
-      var items = _props2.items;
-      var pause = _props2.pause;
-      var eraseMode = _props2.eraseMode;
-      var typingInterval = _props2.typingInterval;
-
-      var other = _objectWithoutProperties(_props2, ['color', 'cursor', 'deletingInterval', 'emptyPause', 'items', 'pause', 'eraseMode', 'typingInterval']);
+      var _props3 = this.props;
```
Comment (charj) [inline lib/ReactRotatingText.js]: Is this change necessary?

File: lib/ReactRotatingText.js (hunk 1 above)
Comment (woofers) [reply lib/ReactRotatingText.js]: @charj I believe these are updates from previously merged PRs however the lib was never re-built.
See https://github.com/adrianmcli/react-rotating-text/pull/14/files

File: lib/ReactRotatingText.js (hunk 2 above)
Comment (woofers) [reply lib/ReactRotatingText.js]: @charj Same comment as above

Comment (charj) [reply lib/ReactRotatingText.js]: Therefore the lib needs rebuilding before this PR can be merged 👍

Comment (woofers) [reply lib/ReactRotatingText.js]: @charj Sounds good.  I merged the latest changes and re-built.

**Comment by adrianmcli** [comment]
Thanks @woofers and @charj!
